eng/Versions.props

```diff
@@ -78,7 +78,8 @@
   <PropertyGroup>
     <!-- Dependencies from https://github.com/dotnet/core-setup -->
     <MicrosoftNETCoreAppRefPackageVersion>3.1.0</MicrosoftNETCoreAppRefPackageVersion>
-    <MicrosoftNETCoreAppRuntimewinx64PackageVersion>3.1.0</MicrosoftNETCoreAppRuntimewinx64PackageVersion>
+    <MicrosoftNETCoreAppInternalPackageVersion>3.1.2-servicing.20064.3</MicrosoftNETCoreAppInternalPackageVersion>
```

Comment:
This should be removed, since there is no eng/Version.Details.xml counter part change (and the build didn't break when we missing it). And if we really missed this change aka.ms/bar will show something like "behind several builds"

Reply:
maybe not

Reply:
let's dig a bit together

Reply:
@wtgodbe do you have any guidance here? Is this the bug in darc?

Reply:
Adding `MicrosoftNETCoreAppInternalPackageVersion` to versions.props was an intentional change in order to download the runtime from the correct location in blob storage - see the corresponding change below in global.json. There should be a corresponding change in versions.details.xml, though. My guess is something got screwed up in the merge. CC @mmitche